codelist-manager- Stop URL templates being escaped by CSV2RDF

robons opened the pull request:
Issue #258 - Adding upgrade to codelist-manager so that it ensures that all codelist valueUrl/propertyUrl/aboutUrl properties do not escape the value found inside the notation/parent_nodation columns.

ajtucker commented:
Looks grand, thanks.

I can see the new scenario checks that the `+`s are in there, but does it check with some input like `st/uff`?

robons commented:
> Looks grand, thanks.
> 
> I can see the new scenario checks that the `+`s are in there, but does it check with some input like `st/uff`?

It's only looking at the metadata json file, so it doesn't check the actual values from the CSV file at all.

ajtucker commented:
> It's only looking at the metadata json file, so it doesn't check the actual values from the CSV file at all.

Ok. Is there a quick way to write a failing test that is then fixed by this? I.e. the test should check that the thing that wasn't working properly now works properly.

robons commented:
> Ok. Is there a quick way to write a failing test that is then fixed by this? I.e. the test should check that the thing that wasn't working properly now works properly.

I've already added a test ensuring that the output from the codelist-manager tool is correct. That's a new test I've added in this PR. I don't really want to be testing CSV2RDF as well here.
